src/chief.t.sol

```diff
@@ -176,42 +181,35 @@ contract DSChiefTest is DSThing, DSTest {
         chief.launch();
     }
 
-    function testFail_enable_system() public {
+    function test_enable_system() public {
         address[] memory slate = new address[](1);
         slate[0] = address(0);
         gov.approve(address(chief), 100000 ether);
         chief.lock(100000 ether - 1);
         chief.vote(slate);
-        chief.launch();
+
+        assertTrue(!try_launch());
+        chief.lock(1);
+        assertTrue( try_launch());
     }
 
-    function testFail_lift_system_before_launch() public {
+    function test_lift_system_before_launch() public {
         address[] memory slate = new address[](1);
-        address nonzero_slate = address(1);
-        slate[0] = nonzero_slate;
+        address    zero_address = address(0);
+        address nonzero_address = address(1);
+        slate[0] = nonzero_address;
         gov.approve(address(chief), 100000 ether);
         chief.lock(100000 ether);
         chief.vote(slate);
-        chief.lift(nonzero_slate);
-        assertEq(chief.hat(), nonzero_slate);
-        chief.launch(); // chief.launch() can't be launched when hat is nonzero
-    }
+        chief.lift(nonzero_address);
+        assertEq(chief.hat(), nonzero_address);
```

Comment:
We might want to add `assertTrue(!chief.isUserRoot(nonzero_address));` here too.